config/configtelemetry/configtelemetry.go

```diff
@@ -106,14 +144,14 @@ func DefaultTelemetrySetting() TelemetrySetting {
 }
 
 // GetMetricsLevel returns the parsed level, or error if unknown value.
-// Empty string is consider unknown value.
+// Empty string is considered an unknown value.
 func (ts TelemetrySetting) GetMetricsLevel() (Level, error) {
-	return parseLevel(ts.MetricsLevelStr)
+	return ParseLevel(ts.MetricsLevelStr)
 }
 
 // ParseLevel returns the Level represented by the string. The parsing is case-insensitive
 // and it returns error if the string value is unknown.
-func parseLevel(str string) (Level, error) {
+func ParseLevel(str string) (Level, error) {
```

Comment:
Why was this made public?

Reply:
I think a prior revision had it used in the unmarshaller.  That is now handled through `UnmarshalText()` and I'll revert this change.